workflow/rules/frag_to_tagAlign.smk

```diff
@@ -0,0 +1,47 @@
+## Convert fragment file to tagAlign file
+rule frag_to_tagAlign:
+	input:
+		frag_file = 
+			os.path.join(
+				config["results_dir"], 
+				"{cluster}", 
+				"Fragments", 
+				"atac_fragments.tsv.gz"
+			)
+	params:
+		chrom_sizes = encode_e2g.ABC.config['ref']['chrom_sizes']
+	output:
+		tagAlign_sort_file = 
+			os.path.join(
+				config["results_dir"], 
+				"{cluster}", 
+				"Fragments",
+				"tagAlign.sort.gz"
+			)
+	conda:
+		os.path.join(
+			config["encode_re2g_dir"],
+			encode_e2g.config["ABC_DIR_PATH"],
+			"workflow",
+			"envs",
+			"abcenv.yml"
+		)
```

Comment:
It'd be better if what requirements we needed from the abcenv.yml file into sc-E2G's conda env yml file.